File: atv-topico-3-4/topico-3-js/gangorra.js
const gangorra = (a1, a2, b1, b2) => {
  return a1 * a2 == b1 * b2 ? 0 : a1 * a2 > b1 * b2 ? -1 : 1;
};
console.log(gangorra(35, 80, 35, 75));

File: atv-topico-3-4/topico-3/gangorra.hs
gangorra e f g h = 
  if e * f == g * h
    then 0
    else if e * f < g * h
      then 1
      else -1

main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    d <- readLn :: IO Int
    print $ gangorra a b c d